HOWTO-it.md

```diff
@@ -0,0 +1,17 @@
+Leggilo in altre lingue: [English](HOWTO.md), [Français](HOWTO-fr.md), [Español](HOWTO-es.md), [Italiano](HOWTO-it.md), [简体中文](HOWTO-zh.md), [हिन्दी](HOWTO-hi.md), [繁體中文](HOWTO-zh_TW.md), [Português (BR)](HOWTO-pt_BR.md), [فارسی](HOWTO-fa_IR.md), [Русский](HOWTO-ru.md)
+
+Benvenuto su Free-Programming-Books! Diamo il benvenuto ai nuovi collaboratori; anche a quelli che fanno la loro prima pull request su Github. Se sei uno di quelli, ecco qualche risorsa che potrebbe aiutarti:
+
+* [Riguardante le Pull Requests](https://help.github.com/articles/about-pull-requests/)
+* [Creare una pull request](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)
+* [Github Hello World](https://guides.github.com/activities/hello-world/)
+* [Youtube - Github Tutorial per Principianti](https://www.youtube.com/watch?v=0fKg7e37bQE)
+* [Youtube - Come forkare una Repository Github e Inviare una Pull Request](https://www.youtube.com/watch?v=G1I3HF4YWEw)
+* [Youtube - Corso accelerato di Markdown](https://www.youtube.com/watch?v=HUBNt18RFbo)
+
+
+Non esitare a fare domande; ogni collaboratore iniziato con una prima pull request (PR). Potresti essere il nostro millesimo collaboratore.
+
+Anche se sei un collaboratore esperto in ambito open source, queste sono cose che potrebbero crearti dei problemi. Una volta che hai inviato la tua PR, Github Actions  avvierà un linter, spesso trovando piccoli problemi di spaziatura o di ordinamento alfabetico. Se ottieni una spunta verde, tutto è pronto per una revisione, ma se non è così clicca su "Details" sotto il check che è stato fallito, analizza l'errore, risolvilo e aggiungi un commit alla PR. 
+
+In fine, se non sei sicuro che la risorsa che vuoi aggiungere è appropiata a Free-Programming-Books, leggi le linee guida su [CONTRIBUTING](CONTRIBUTING.md) (inglese).
```

Comment:
Link CONTRIBUTING with the localized resource that you also provide :stuck_out_tongue_winking_eye: 

```markdown
In fine, se non sei sicuro che la risorsa che vuoi aggiungere è appropiata a Free-Programming-Books, leggi le linee guida su [CONTRIBUTING](CONTRIBUTING-it.md).
```